tests/test_core.py

```diff
@@ -1364,12 +1382,11 @@ def test_exceptions_white_list(self):
 
     if not self.is_wasm():
       print(size, empty_size, fake_size, disabled_size)
-
-      assert size - empty_size > 0.0025 * size, [empty_size, size]  # big change when we disable entirely
-      assert size - fake_size > 0.0025 * size, [fake_size, size]
-      assert abs(empty_size - fake_size) < 0.007 * size, [empty_size, fake_size]
-      assert empty_size - disabled_size < 0.007 * size, [empty_size, disabled_size]  # full disable removes a little bit more
-      assert fake_size - disabled_size < 0.007 * size, [disabled_size, fake_size]
+      assert empty_size == fake_size, [empty_size, fake_size]
+      # big change when we disable exception catching of the function
+      assert size - empty_size > 0.01 * size, [empty_size, size]
+      # full disable can remove a little bit more
+      assert empty_size >= disabled_size, [empty_size, disabled_size]
```

Comment:
I simplified the code size testing criteria here
- I don't see any reason why `empty_size` and `fake_size` can be different
- Here in some case we are comparing wasm files and in others js files, so I don't think there is any meaningful single value for size differences. And I noticed `empty_size` and `disabled_size` are the same in `wasm2jsN` where `N >= 1`. I guess optimizers remove unused things or something.